news/fixCommandsCacheGen.rst

```diff
@@ -0,0 +1,7 @@
+**Changed:**
```

Comment:
For the news entry file, you should leave the unused headers alone (along with their trailing `None`).  items under a header should have a leading `*` followed by a single space before any text